src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+       href="#logo"
+       class="logo"
+      >
+        <img
+          src="images/logo.png"
+          class="logo_image"
+          alt="Moyo Company Logo"
```

Comment:
The `alt` attribute should describe the image content. `Moyo Company Logo` is a good start, but it would be better to add more specific details about the logo, if possible.